packages/design-system/src/components/contextMenu/menu.js

```diff
@@ -172,6 +172,19 @@ MenuList.propTypes = {
   isIconMenu: PropTypes.bool,
 };
 
+/**
+ * Default wrapper to wrap the context menu's menu item. Necessary to allow
+ * other apps the ability to inject their own media pickers.
+ *
+ * @param {Object} props Component props
+ * @param {Function} props.children Children as a function.
```

Comment:
```suggestion
 * @param {Function} props.render Decorating wrapper function for menu item components.
```

Rename `children` to `render`? To avoid confusion with the normal type of React children, and "render prop" is an [established convention](https://reactjs.org/docs/render-props.html).

Also stumbled across this: https://americanexpress.io/faccs-are-an-antipattern/